### PR TITLE
config: allow parsing bool in --var

### PR DESCRIPTION
**Pull request by nirmoy**
## What does this PR change?

before the PR
I0108 17:33:27.004243   32429 config.go:304] [kubic] setting 'Network.MultipleCni'='true'
cannot write string to field MultipleCni(bool) : true

## Documentation
- No documentation needed

### CODE REVIEW

File: pkg/config/config.go [hunk 1]
```diff
@@ -293,15 +294,20 @@ func (kubicCfg *KubicInitConfiguration) ToConfigMap(client clientset.Interface,
...
 			}
 
 			glog.V(8).Infof("[kubic] setting '%s'='%s'", components[0], components[1])
-			err = swalker.Write(components[0], kubicCfg, components[1])
+			b, err := strconv.ParseBool(components[1])
+			if err != nil {
+				err = swalker.Write(components[0], kubicCfg, components[1])
+			} else {
+				err = swalker.Write(components[0], kubicCfg, b)
+
```
Comment (inercia) [inline pkg/config/config.go]:
```suggestion
```

File: pkg/config/config.go [hunk 2]
```diff
@@ -293,15 +294,20 @@ func (kubicCfg *KubicInitConfiguration) ToConfigMap(client clientset.Interface,
...
-		var err error
 		for _, v := range vars {
 			components := strings.Split(v, "=")
 			if len(components) != 2 {
 				return fmt.Errorf("cannot parse '%s' as an assignment", v)
 			}
 
 			glog.V(8).Infof("[kubic] setting '%s'='%s'", components[0], components[1])
-			err = swalker.Write(components[0], kubicCfg, components[1])
+			b, err := strconv.ParseBool(components[1])
```
Comment (MalloZup) [inline pkg/config/config.go]: `nit` can we have more then `b` name? imho would be more readable

Comment (inercia) [reply pkg/config/config.go]: I agree that it would be better, but not a biig deal in my view...

Comment (nirmoy) [reply pkg/config/config.go]: @MalloZup Thanks,  done!

**Pull request merged**